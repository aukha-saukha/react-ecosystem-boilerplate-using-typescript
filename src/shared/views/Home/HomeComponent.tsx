import * as React from 'react';

const HomeComponent: React.FC<{}> = (): JSX.Element => (
  <main className="font-size-16" data-test="main">
    Home page
  </main>
);

export default HomeComponent;
