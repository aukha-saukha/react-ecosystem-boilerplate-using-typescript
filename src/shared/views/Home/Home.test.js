import { shallow } from 'enzyme';
import React from 'react';

import Home from './HomeComponent';

describe('Home component', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(<Home />);

    expect(wrapper).toMatchSnapshot();
  });
});
