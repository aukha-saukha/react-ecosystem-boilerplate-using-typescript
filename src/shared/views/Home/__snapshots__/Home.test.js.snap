// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component Renders correctly 1`] = `
<main
  className="font-size-16"
  data-test="main"
>
  Home page
</main>
`;
